NETWORK/NetworkGetNetworkIdFromEntity.md

```diff
@@ -7,9 +7,31 @@ ns: NETWORK
 // 0xA11700682F3AD45C 0x9E35DAB6
 int NETWORK_GET_NETWORK_ID_FROM_ENTITY(Entity entity);
 ```
+Get the network id of the given entity
 
+An entity network id is representing a given entity for other clients, is like a handle that every client recognize, can be reconverted to a client entity handle. 
+These can and will be reused, the network id will not change.
+If you need to refer to an entity across machines (clients, or the server), you should use its network id.
 
+Read more at [Network and local IDs](https://docs.fivem.net/docs/scripting-manual/networking/ids/)
 ## Parameters
-* **entity**: 
+* **entity**: the handle of the entity from which the network id should be taken
 
 ## Return value
+The network id of the given entity
+
+## Examples
+```lua
+local obj = CreateObject(`prop_weed_01`, vector3(0.0, 0.0, 0.0), true)
+local netId = NetworkGetNetworkIdFromEntity(obj)
+```
+
+```js  
+const obj = CreateObject(GetHashKey("prop_weed_01"), vector3(0.0, 0.0, 0.0), true)
+const netId = NetworkGetNetworkIdFromEntity(obj)
+```
+
+```cs  
+const obj = CreateObject(GetHashKey("prop_weed_01"), vector3(0.0f, 0.0f, 0.0f), true)
```

Comment:
Small detail, `const` can't be used in this case because `CreateObject` doesn't return a compile-time constant value.

Reply:
i have programmed very little in c#, thanks for the correction!

Reply:
Also the `vector3` should be `new Vector3` ;)